### ISSUE TITLE
adjust text link style in rich text (all content items): 3

### ISSUE THREAD

**Issue by paregorios** [isawnyu/isaw.web#353]
Presently, our links get a distinctive color and, on mouseover, a solid underline. This is insufficient for individuals with color perception difficulties. Change CSS directives so that:

- Text links in all rich text contexts have solid underline (regular and visited; **all element contexts other than nav and button-like links**)
- On focus and mouseover: underline becomes dashed, text-decoration-color and text color become "blue"
- On active, text-decoration becomes "none", text color becomes "orange", and background color becomes "black"

I believe that the following CSS directives will handle the base style:

```css
text-decoration-color: #66262d;
text-decoration-line: underline;
text-decoration-style: solid;
```

Appropriate values and event selectors should be used to achieve the other states described above.

**Comment by paregorios** [comment]
So, this is what I want to see for headings (any level) inside main content and inside a link:

initial/plain/visited:

![screen shot 2018-12-03 at 9 08 49 am](https://user-images.githubusercontent.com/263285/49382121-298a4b80-f6db-11e8-8ffb-31380f497c0e.png)

focus/mouseover:

![screen shot 2018-12-03 at 9 08 27 am](https://user-images.githubusercontent.com/263285/49382133-33ac4a00-f6db-11e8-8b22-a509f3bf841d.png)

active:
![screen shot 2018-12-03 at 9 07 30 am](https://user-images.githubusercontent.com/263285/49382150-41fa6600-f6db-11e8-9069-92d557607a5d.png)

**Comment by paregorios** [comment]
Here are examples for links in paragraph text.

initial/plain/visited:

![screen shot 2018-12-03 at 9 12 51 am](https://user-images.githubusercontent.com/263285/49382399-f09ea680-f6db-11e8-8205-38315fac267b.png)

focus/mouseover:

![screen shot 2018-12-03 at 9 13 54 am](https://user-images.githubusercontent.com/263285/49382521-3ce9e680-f6dc-11e8-93ae-c61aacafeca8.png)

active:

![screen shot 2018-12-03 at 9 14 24 am](https://user-images.githubusercontent.com/263285/49382542-4a06d580-f6dc-11e8-8fc0-80688739bfc4.png)

**Comment by alecpm** [comment]
This is up on staging for review.

**Comment by k2soule** [comment]
This is mostly working on staging - we're just seeing one thing that still needs to change: The text color needs to change to blue on focus/mouse-over.

**Comment by paregorios** [comment]
The text color also needs to change to orange on :active: (which one can test by click-holding on the link).

**Comment by alecpm** [comment]
Needed a few `!important`s to get the colors set on some views. Should be working now.
